### ISSUE TITLE
I'm not able to publish my model to Github

### ISSUE THREAD

**Issue by obernauerhof** [archimatetool/archi-modelrepository-plugin#209]
### Version of Archi, Operating System
5.0.0 on MacOs and 5.0.1 on Windows

### Version of this plug-in
0.8.7.202212021

### Any other plug-ins installed?
jarchi, excel, lightbox

### Expected Behaviour
published, uploaded xml files
I guess on the local repository model-repository\archimate\.git\temp.archimate should be filled with xml files I guess, it's empty

### Actual Behaviour
New Model Repository - Error message:
There was and error: https://github.com/...e.git: not authorized
I also used without .git
I'm able to write to the repository, from web and treeview - I'm the admin

### Steps to Reproduce the Behaviour
Menu Collaboration - Add local Model to Workplace and Publish, same with Publish

But so far the only really concern / issue I'm facing, really great work, thx Phillip and Jean-Baptiste!

**Comment by obernauerhof** [comment]
Dear Phillipus,
Great, very fast reply from your end, seems, I'm late now :-).
I didn't wish to publish the full URL, was wondering, if someone shouts at me, I don't care about privacy ... Therefore I replaced with dots the full URL. But frankly said, I doubt, this would be the case with an empty repository, so here we go, it's a private one. But I also used an enterprise repository, I asked the engineering team @Bergfreunde to provide some, to be sure, I don't do sth wrong as a non developer ... Same error message and behaviour.
https://github.com/obernauerhof/bergfreunde
https://github.com/bergfreunde/archimate
Cheers, Alexander
